File: 2024.1-Exercícios/Aula-6/questao2.c
/* Escreva um programa em C que solicita ao usuário uma frase e converte todas as letras maiúsculas em minúsculas e todas as letras minúsculas em maiúsculas. 
O programa deve imprimir a frase resultante. */

#include <stdio.h>
#include <string.h>

int main() {
    char str[100];
    int i;

    printf("Digite uma string: ");
    gets(str);

    for (i = 0; str[i] != '\0'; i++) {
        if (str[i] >= 'A' && str[i] <= 'Z') {
            str[i] += 32;
        } else if (str[i] >= 'a' && str[i] <= 'z') {
            str[i] -= 32;
        }
    }

    printf("String resultante: %s\n", str);

    return 0;
}

File: 2024.1-Exercícios/Aula-14/questao2.py
"""Escreva um programa em Python que leia o arquivo alunos.txt (criado na questão anterior) e permita ao usuário buscar informações de um aluno pelo nome. 
O programa deve exibir o nome, a idade e a nota do aluno, caso o aluno seja encontrado no arquivo. Se o nome do aluno não for encontrado, o programa deve exibir uma mensagem informando que o aluno não foi encontrado.
Requisitos:
● O programa deve solicitar ao usuário o nome do aluno a ser pesquisado.
● Se o aluno for encontrado, o programa deve exibir todas as informações relacionadas a ele (nome, idade, nota).
● Caso o nome não seja encontrado no arquivo, exiba a mensagem: "Aluno não encontrado."."""

nome = input("Digite o nome do aluno a ser pesquisado: ")
with open("alunos.txt", "r") as file:
    alunos = file.read().split("---\n")
    for aluno in alunos:
        if nome in aluno:
            print(aluno)
            break
    else:
        print("Aluno não encontrado.")